packages/quantic/force-app/main/default/lwc/exampleSearch/exampleSearch.html

```diff
@@ -3,7 +3,7 @@
         <c-quantic-search-interface engine-id={engineId}>
             <div class="slds-grid slds-gutters slds-wrap main slds-grid_align-center">
                 <div class="slds-col slds-order_2 slds-large-order_1 slds-size_1-of-1 slds-large-size_4-of-12">
-                    <c-quantic-facet field="objecttype" label="Type" engine-id={engineId}></c-quantic-facet>
+                    <c-quantic-facet field="objecttype" label="Type" engine-id={engineId} data-cy="type"></c-quantic-facet>
```

Comment:
Why is this necessary just on one facet?

Reply:
Because the full search tests are only targeting this facet at this moment. When adding tests referencing other facets we'd need to add the attribute for them as well.

Reply:
I'm a bit concerned that having this in the exampleSearch will confuse users relying on this to learn how to implement our components.

Reply:
Unfortunately, this is necessary for identifying the right facet. But I understand your concern and that's one of the reasons why I prefer to have test-oriented components in a separate package (as shown in [this PR](https://github.com/coveo/ui-kit/pull/1027)).

Reply:
Oh i see! Once we use a community instead then we aren't bound by the localdev limitation. Makes sense thanks!